src/detections/context.js

```diff
@@ -1,12 +1,16 @@
 import React, { useContext as useReactContext, useState } from 'react'
+import { useFormik } from 'formik'
+
+import { useSetFormik } from 'detections/hooks/use-set-formik'
 
 export const Context = React.createContext({})
 
 export const useContext = () => useReactContext(Context)
 
 export const useDefaultContext = () => {
-  const [pageIndex, setPageIndex] = useState(0)
   const [pageSize, setPageSize] = useState(10)
+  const [pageIndex, setPageIndex] = useState(0)
+  const formik = useFormik({ ...useSetFormik(pageSize, pageIndex) })
```

Comment:
No need for a new object here, since such an object is already returned by a function call.